diff -Nur bucardo/bucardo.schema bucardo.new/bucardo.schema
--- bucardo/bucardo.schema	2013-01-08 13:17:52.000000000 +0100
+++ bucardo.new/bucardo.schema	2013-01-08 13:29:17.968256714 +0100
@@ -136,9 +136,9 @@
 log_showlevel|0|Show log level in the log output?
 log_showline|0|Show line number in the log output?
 log_showtime|3|Show timestamp in the log output?  0=off  1=seconds since epoch  2=scalar gmtime  3=scalar localtime
-log_conflict_file|bucardo_conflict.log|Name of the conflict detail log file
+log_conflict_file|/var/log/bucardo/conflict.log|Name of the conflict detail log file
 log_level|NORMAL|How verbose to make the logging. Higher is more verbose.
-warning_file|bucardo.warning.log|File containing all log lines starting with "Warning"
+warning_file|/var/local/bucardo/warning.log|File containing all log lines starting with "Warning"
 \.
 
 -- Versioning
@@ -155,8 +155,8 @@
 FROM STDIN
 WITH DELIMITER '|';
 bucardo_vac|1|Do we want the automatic VAC daemon to run?
-default_email_from|nobody@example.com|Who the alert emails are sent as
-default_email_to|nobody@example.com|Who to send alert emails to
+default_email_from|root|Who the alert emails are sent as
+default_email_to|root|Who to send alert emails to
 default_email_host|localhost|Which host to send email through
 default_conflict_strategy|bucardo_latest|Default conflict strategy for all syncs
 email_debug_file||File to save a copy of all outgoing emails to
@@ -164,7 +164,7 @@
 host_safety_check||Regex to make sure we don't accidentally run where we should not
 isolation_level|repeatable read|Default isolation level: can be serializable or repeatable read
 piddir|/var/run/bucardo|Directory holding Bucardo PID files
-reason_file|bucardo.restart.reason.txt|File to hold reasons for stopping and starting
+reason_file|/var/log/bucardo/restart.reason|File to hold reasons for stopping and starting
 semaphore_table|bucardo_status|Table to let apps know a sync is ongoing
 statement_chunk_size|10000|How many primary keys to shove into a single statement
 stats_script_url|http://www.bucardo.org/|Location of the stats script
