diff -Nur bucardo/bucardo.schema bucardo.new/bucardo.schema
--- bucardo/bucardo.schema	2011-06-21 14:42:32.673483387 +1000
+++ bucardo.new/bucardo.schema	2011-06-21 14:56:06.745509641 +1000
@@ -12,7 +12,8 @@
 ALTER USER bucardo SET search_path = bucardo;
 SET CLIENT_MIN_MESSAGES = 'WARNING';
 
-\c bucardo bucardo
+\c bucardo
+SET SESSION AUTHORIZATION bucardo;
 
 -- plpgsql and plperlu are loaded, but just in case:
 SET client_min_messages = 'FATAL';
